eng/testing/tests.mobile.targets

```diff
@@ -54,7 +54,7 @@
              AssemblyFile="$(RuntimeConfigParserTasksAssemblyPath)"
              Condition="'$(RuntimeConfigParserTasksAssemblyPath)' != ''" />
 
-  <Target Name="GenerateRuntimeConfig" Condition="'$(OutputType)' != 'library' and '$(TargetOS)' != 'Browser'">
```

Comment:
Out of curiosity, why do we no longer need this, or why was it introduced in the first place?

Reply:
I think it was introduced by mistake from https://github.com/dotnet/runtime/pull/56486

Reply:
why are we skipping it on browser

Reply:
Good question.  @fanyang-mono ?

Reply:
Wasm has its own target of generating runtime config which is part of the source code. (https://github.com/dotnet/runtime/blob/main/src/mono/wasm/build/WasmApp.targets#L146) This target is for mobile testing only. But for wasm testing, the targets defined in this file will be visible to wasm tests as well. To make things clear, I added `'$(TargetOS)' != 'Browser'`.